src/main/java/com/pnu/dev/radioserviceapi/controller/api/VideoApiController.java

```diff
@@ -1,15 +1,38 @@
 package com.pnu.dev.radioserviceapi.controller.api;
 
+import com.pnu.dev.radioserviceapi.dto.response.PageResponse;
+import com.pnu.dev.radioserviceapi.dto.response.VideoDto;
+import com.pnu.dev.radioserviceapi.dto.response.VideosCollectionResponse;
+import com.pnu.dev.radioserviceapi.service.VideoApiService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 @RequestMapping("/api/v1/videos")
 public class VideoApiController {
 
-    // DTO with :
-    // 1. recent - list of recent videos (call out to youtube api)
-    // 2. recommended - list of recommended videos
-    // 3. planned or current streams
+
+    private final VideoApiService videoApiService;
+
+    @Autowired
+    public VideoApiController(VideoApiService videoApiService) {
+        this.videoApiService = videoApiService;
+    }
+
+    @GetMapping("/recommended")
+    public PageResponse<VideoDto> findRecommended(@PageableDefault(size = 10)
```

Comment:
Maybe it's better to create separate dto for recommended video, becase `VideoDto` will always have two null fields?